readthedocs/proxito/tests/responses/v0.json

```diff
@@ -0,0 +1,120 @@
+{
+    "comment": "THIS RESPONSE IS IN ALPHA FOR TEST PURPOSES ONLY AND IT'S GOING TO CHANGE COMPLETELY -- DO NOT USE IT!",
+            "project": {
+                "created": "2019-04-29T10:00:00Z",
+                "default_branch": "master",
+                "default_version": "latest",
+                "id": 1,
+                "language": {
+                    "code": "en",
+                    "name": "English"
+                },
+                "modified": "2019-04-29T12:00:00Z",
+                "name": "project",
+                "programming_language": {
+                    "code": "words",
+                    "name": "Only Words"
+                },
+                "homepage": "http://project.com",
+                "repository": {
+                    "type": "git",
+                    "url": "https://github.com/readthedocs/project"
+                },
+                "slug": "project",
+                "subproject_of": null,
+                "tags": [
+                    "project",
+                    "tag",
+                    "test"
+                ],
+                "translation_of": null,
+                "users": [
+                    {
+                    "username": "testuser"
+                }
+                ]
+            },
+            "version": {
+                "active": true,
+                "hidden": false,
+                "built": true,
+                "downloads": {},
+                "id": 1,
+                "identifier": "a1b2c3",
+                "ref": null,
+                "slug": "latest",
+                "type": "tag",
+                "verbose_name": "latest"
+            },
+            "build": {
+                "commit": "a1b2c3",
+                "created": "2019-04-29T10:00:00Z",
+                "duration": 60,
+                "error": "",
+                "finished": "2019-04-29T10:01:00Z",
+                "id": 1,
+                "project": "project",
+                "state": {
+                    "code": "finished",
+                    "name": "Finished"
+                },
```

Comment:
This is probably commentary on the build serializer and not this PR, but I found myself wanting some helpers here around the build state data. The data is all here, but I had to replicate a good deal of backend logic to determine things like `is_finished`, `is_cancelled`, `is_active`, etc.

Consider this a side request I supposed :slightly_smiling_face:

Reply:
For example, I had to sort through a lot of build logic with the API response this the build detail view, for example: https://github.com/readthedocs/ext-theme/blob/main/src/js/build/detail.js#L236-L283

Reply:
> The data is all here, but I had to replicate a good deal of backend logic to determine things like is_finished, is_cancelled, is_active, etc.

This is a pretty common discussion between backend and frontend teams 😄 . IMHO, the data is there and it's the FE who should create "the helpers" required to present the data as they need --but I guess this is kind of a philosophical discussion. We can continue talking about this outside this PR.

In any case, if we decide changing anything here, it should be done in the `BuildSerializer`.